qtpy/tests/utils.py

```diff
@@ -0,0 +1,11 @@
+"""Utility functions for tests."""
+
+import os
+
+
+def using_conda():
+    return os.environ.get('USE_CONDA', 'Yes') == 'Yes'
+
+
+def not_using_conda():
+    return os.environ.get('USE_CONDA', 'No') == 'No'
```

Comment:
I don't know if this is really necessary because it's equivalent to `not using_conda()`, which is simpler to read in my opinion.

@CAM-Gerlach, what do you think?

Reply:
Actually, its not, because `not using_conda()` would return falsey if `USE_CONDA` is unset, as it would be anywhere outside of the CI, and thus the tests wouldn't skip as they would with `not_using_conda(). However, the fact that both return truthy if it is not set is potentially confusing. If you wanted to make it one function, you could have it return `True` if `Yes`, `False` if `No` and `None` if undefined, but this makes using it more tedious and potentially error-prone, as much or more so than not using a function at all.

Ideally, we could actually determine from the environment if the package was installed using conda or not, but all the ways I explored to do that were complex, hacky and unreliable; maybe either of you know a better way.

Reply:
Ok, I understand now. So we also need to leave this as it is.